src/main/java/frc/team2412/robot/subsystems/DrivebaseSubsystem.java

```diff
@@ -57,15 +89,12 @@ public DrivebaseSubsystem() {
 			throw new RuntimeException();
 		}
 
-		// set drive motors to brake
 		swerveDrive.setMotorIdleMode(true);
```

Comment:
(no action required) I do wish swervelib renamed this method or added an enum to make it clear that `true` means brake mode...